src/css/components/panel.scss

```diff
@@ -40,6 +40,18 @@ $panel-padding: $grid-3;
   flex-direction: row;
   justify-content: flex-start;
   margin: 0;
+
+  p, span {
+    font-size: 0.9rem;
+  }
+}
+
+* + .panel-actions {
+  margin-top: $grid-1;
+}
+
+.panel-actions + * {
+  margin-top: $grid-2;
```

Comment:
Isn't the structure of panels fairly known that we don't really need these `*` rules? I mean, we could say something like `.panel-documentation + .panel-actions` rules instead?